candy-machine/program/tests/core/helpers.rs

```diff
@@ -0,0 +1,313 @@
+use std::str::FromStr;
```

Comment:
I like those helpers.
Maybe it's time to pull those into a separate crate we can use all over? 
Maybe `sol-test-utils` or something like that for the solana core specific ones?
And `mp-test-utils` for the ones involving metaplex programs?